packages/shared/components/modals/ProposalDetailsMenu.svelte

```diff
@@ -18,11 +32,19 @@
 
 <Modal bind:this={modal} {...$$restProps}>
     <div class="flex flex-col">
+        {#if _isVotingForProposal}
+            <MenuItem
+                icon={Icon.Minus}
```

Comment:
This icon doesn't correspond with the designs

Reply:
Right, I meant to update this, but also couldn't find the icon that we needed.